src/Surfnet/Stepup/Tests/Configuration/InstitutionConfigurationTest.php

```diff
@@ -46,6 +49,9 @@ public function use_ra_locations_option_is_set_to_false_by_default_upon_creation
         $verifyEmailOption               = new VerifyEmailOption(true);
         $numberOfTokensPerIdentityOption = new NumberOfTokensPerIdentityOption(0);
         $expectedUseRaLocationsOption = new UseRaLocationsOption(false);
+        $useRaOption = new UseRaOption(null);
+        $useRaaOption = new UseRaaOption(null);
+        $selectRaaOption = new SelectRaaOption(null);
 
         $this->scenario
             ->when(function () use ($institution, $institutionConfigurationId, $showRaaContactInformationOption, $verifyEmailOption) {
```

Comment:
Is it possible to test in the `then` if the sub events are thrown?

Reply:
The then describes the expected outcome of, in this case, the creation of the InstitutionConfiguration aggregate. In the annotated test, the 'sub events (`applyUseRaOptionChangedEvent`)' are not triggered.

Reply:
For that to happen we would need to trigger the `rebuild` method on the aggregate. Which would also be an interesting thing to test in the `InstitutionConfigurationTest`.

Reply:
Testing the rebuild method was out of scope for this test, I've added a test to cover the FGA configure options on the aggregate. See commit ea844e6

Reply:
Thanks for looking into this.